docs/releases/2.2.1.txt

```diff
@@ -25,3 +25,6 @@ Bugfixes
 
 * Fixed a regression in Django 2.2 that caused a single instance fast-delete
   to not set the primary key to ``None`` (:ticket:`30330`).
+
+* Fixed an issue with the auto-reloader not detecting changes
+  on Windows (:ticket:`30323`).
```

Comment:
I think this description doesn't completely cover changes that we've made, e.g. catching exceptions from `get_resolver().urlconf_module` is not a Windows-fix, IMO.

Maybe we should also point reloader? e.g.
```
Fixed a regression in Django 2.2 where ...
when using ``StatReloader`` on Windows (:ticket:`30323`).
```

Reply:
I've improved the message, let me know what you think.